tests/climate-change-v2/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
```

Comment:
Add a `set -eo pipefail` in case anything breaks

Reply:
Done.

A remaining issue is that we need to check that the Jenkins build doesn't fail if any tests fail, as Jenkins does a bit of clean up (closing down the Docker containers).

At the same time, test failures should be reported up to GitHub and stop us from merging failing builds.

Reply:
In another Jenkinsfile, I have 
```
       stage('run') {
            steps {
                catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') { 
                         // something to run the tests
                }
            }
        }
```

I've not used the github integration so I'm not sure how this interacts with the github plugin; maybe easiest to just try using that fragment?

And move the `                         sh "docker-compose down"` step out of the  End User Tests stage into the `post { always { `

Reply:
Thanks @nosnickid. Good point about putting `docker-compose down` in the `post { always {` bit -- done.

For the current climate-change site, any test failures percolate all the way back up to GitHub.

In this case, the `run.sh` script will return the exit code from Cucumber and the Docker container returns this to the Jenkins stage which should fail and the eventual junit plugin reports this via GitHub Checks API.